Documentation/diff-generate-patch.txt

```diff
@@ -74,6 +74,11 @@ separate lines indicate the old and the new mode.
       rename from b
       rename to a
 
+5.  Hunk headers mention the name of the function to which the hunk
+    applies.  See "Defining a custom hunk-header" in
+    linkgit:gitattributes[5] for details of how to tailor to this to
```

Comment:
I doubt @gitgitgadget will like this, but there are too many `to`s here.
```suggestion
    linkgit:gitattributes[5] for details of how to tailor this to
```

Reply:
> I doubt @gitgitgadget will like this

Let's find out!  Thanks for spotting this.

Reply:
In theory, iiuc (and I have less experience than you do!), you should squash the correction into your original commit, and then edit the summary to explain the additional change.

You can have a signed-off-by from me :-).